docs/iris/src/whatsnew/contributions_1.10/newfeature_2015-Jan-21_allow-concat-of-cubes-with-anon-dims.txt

```diff
@@ -1 +1 @@
-* Cubes with anonymous dimensions can now be concatenated.
+* Cubes with anonymous dimensions can now be concatenated. This can only occur along a dimension that is not anonymous.
```

Comment:
Really this comment should be part of the scope of the concatenation method(s) as well.